src/devices/README.md

```diff
@@ -328,6 +328,7 @@ Our vision: the majority of .NET bindings are written completely in .NET languag
 ### Weight sensors
 
 * [DFRobot KIT0176: I2C 1kg Weight Sensor Kit - HX711 (Gravity: I2CWeight Sensor)](Hx711I2c/README.md)
+* [HX711: Load Cell amplifier and 24-Bit ADC Module](Hx711/README.md)
```

Comment:
that's not needed, it's generated automatically. What's needed is the category file to handle this automatically :-)

Reply:
Ok, I'll remove it. The category file means the readme.md? Or do I have to add another file?

Reply:
No, there should be a file called `category.txt` in the folder of your binding (/src/devices/Hx711). The recognized categories are in `/tools/device-listing/program.cs`.

Reply:
Thanks, i add this.